src/wallet/test/coinselector_tests.cpp

```diff
@@ -300,7 +300,7 @@ BOOST_AUTO_TEST_CASE(bnb_search_test)
         CCoinControl coin_control;
```

Comment:
Any reason to not rename in line 269?

Reply:
Missed it when resolving conflicts. Done.